### ISSUE TITLE
cy.request timeouts reported oddly in the terminal report

### ISSUE THREAD

**Issue by jmoses** [archfz/cypress-terminal-report#89]
```
      {
        "type": "cy:request",
        "severity": "error",
        "message": "POST https://my-url-here.com/foo\nCannot parse cy.request error content!"
      }
```

junit output:

```
Error Message
`cy.request()` timed out waiting `30000ms` for a response from your server.  The request we sent was:  Method: POST URL: https://url-goes-here  No response was received within the timeout.  https://on.cypress.io/request
Stacktrace
CypressError: `cy.request()` timed out waiting `30000ms` for a response from your server.

The request we sent was:

Method: POST
URL: https:/url-goes-here

No response was received within the timeout.

https://on.cypress.io/request
    at http://localhost:43455/__cypress/runner/cypress_runner.js:159435:19
    at tryCatcher (http://localhost:43455/__cypress/runner/cypress_runner.js:10609:23)
    at http://localhost:43455/__cypress/runner/cypress_runner.js:5731:41
    at tryCatcher (http://localhost:43455/__cypress/runner/cypress_runner.js:10609:23)
    at Promise._settlePromiseFromHandler (http://localhost:43455/__cypress/runner/cypress_runner.js:8544:31)
    at Promise._settlePromise (http://localhost:43455/__cypress/runner/cypress_runner.js:8601:18)
    at Promise._settlePromise0 (http://localhost:43455/__cypress/runner/cypress_runner.js:8646:10)
    at Promise._settlePromises (http://localhost:43455/__cypress/runner/cypress_runner.js:8722:18)
    at _drainQueueStep (http://localhost:43455/__cypress/runner/cypress_runner.js:5316:12)
    at _drainQueue (http://localhost:43455/__cypress/runner/cypress_runner.js:5309:9)
    at Async.../../node_modules/bluebird/js/release/async.js.Async._drainQueues (http://localhost:43455/__cypress/runner/cypress_runner.js:5325:5)
    at Async.drainQueues (http://localhost:43455/__cypress/runner/cypress_runner.js:5195:14)
From Your Spec Code:
    at Context.eval (http://localhost:43455/__cypress/tests?p=cypress/support/index.js:782:13)
From Your Spec Code:
    at Context.eval (http://localhost:43455/__cypress/tests?p=cypress/support/index.js:782:13)
```

**Comment by jmoses** [comment]
Oh, yeah, that's what I meant, that the log should just surface the actual cypress error message there, instead of a seemingly unrelated parsing message.

**Comment by archfz** [comment]
Released in 3.1.0.
